### PR TITLE
Staging area for 0.1.0-beta version

### PR DESCRIPTION
**Pull request by LukasBures**
# Arizen features for v0.1.0-beta version
- [x] team GPG verified commits 
- [x] codeclimate code/bug automatic review
- [x] Travis CI integration
- [x] dependency version checker 
- [x] displaying all your addresses with balances
- [x] displaying detail info about address
- [x] transaction history - raw
- [x] transaction history - with detail about transaction
- [x] introduce new `.awd` file (arizen wallet database)
- [x] import wallet.dat - during registration
- [x] import wallet.dat / `[your_login].awd` encrypted / `[your_login].awd` unencrypted - in GUI
- [x] backup wallet to encrypted/decrypted file
- [x] desktop notifications
- [x] light SVP wallet without requiring synchronization - connected to online insight
- [x] use modern cryptography, aes-256-gcm with sha512 key
- [x] encrypted login pass
- [x] encrypted `[your_login].awd` file - storing all your data
- [x] encrypted `users.arizen` file - storing all users login data
- [x] multiple accounts
- [x] password protection 
- [x] you have to create strong password 
- [x] automatically download updates
- [x] simple, nice, flat, ZEN branded GUI
- [x] Windows + MacOS + Linux instalation files
- [x] encrypted customizable wallet setting
- [x] implement settings for selection of your favorite insight explorer
- [x] implement settings for selection of your favorite insight API
- [x] implement settings for enabling/disabling notification
## Wallet
- [x] show/hide zero balance addresses
- [x] sending transparent transaction
- [x] generating new transparent address + label
- [x] configurable fee

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -4,23 +4,52 @@
...
+    loggedIn: false,
+    login: "",
+    pass: "",
+    walletDb: [],
+    dbChanged: false
+};
+
+const dbStructWallet = "CREATE TABLE wallet (id INTEGER PRIMARY KEY AUTOINCREMENT, pk TEXT, addr TEXT UNIQUE, lastbalance REAL, name TEXT);";
+const dbStructContacts = "CREATE TABLE contacts (id INTEGER PRIMARY KEY AUTOINCREMENT, addr TEXT UNIQUE, name TEXT, nick TEXT);";
+const zenApi = "https://explorer.zensystem.io/insight-api-zen/";
```
Comment (aayanl) [inline app/main.js]: maybe have this be a configurable option in case explorer goes down?

File: app/test_data.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// @flow
+/*jshint esversion: 6 */
+/*jslint node: true */
+"use strict";
+
+let walletNames = ["My First Wallet", "My Second Wallet", "New Car", "Messenger Wallet"];
+
+let wallets = [{id : 0, address: "znbcGtStHjZdfWsRhZdZsffaetZZeDfv524", balance: 110.00},
```
Comment (aayanl) [inline app/test_data.js]: Maybe add some multisig addresses here? example:  zsyF68hcYYNLPj5i4PfQJ1kUY6nsFnZkc82

File: app/main.js (hunk 1 above)
Comment (LukasBures) [reply app/main.js]: yes, that is solution too

I think about:
implement random selecton from multiple insight for distribution load over multiple servers

File: app/test_data.js (hunk 2 above)
Comment (LukasBures) [reply app/test_data.js]: Just detail, but added
(that are test data - only for dummy visualization)

**Comment by LukasBures** [comment]
Merging

**Pull request merged**